docs/index.rst

```diff
@@ -61,10 +61,18 @@ Pass along command-line arguments as your program would normally expect them::
 
     ddtrace-run gunicorn myapp.wsgi:application --max-requests 1000 --statsd-host localhost:8125
 
-`For most users, this should be sufficient to see your application traces in Datadog.`
+*As long as your application isn't running in* ``DEBUG`` *mode, this should be enough to see your application traces in Datadog.*
 
-`Please read on if you are curious about further configuration, or
-would rather set up Datadog Tracing explicitly in code.`
+If you're running in a Kubernetes cluster, and still don't see your traces, make sure your application has a route to the tracing Agent. An easy way to test this is with a::
```

Comment:
Would make sense to move this part in a separated section? It's kind of generic and applies to other frameworks too.

Reply:
Sure! If you can find a place for it that makes more sense, I'm cool with that.

I initially thought putting this at the top where we're showing how to set up the Agent made sense. 

But based upon feedback I've been seeing, I think a separate dedicated section for debugging might make more sense. Possibly going into detail about differences between `ddtrace-run` and `patch_all()`, and showing how to enable debugging in code without the environment variable set.

Reply:
What if we create a Troubleshooting section after `Advance Usage`?

Also yes, we need to describe the difference between `ddtrace-run` and `patch_all()` at the beginning of the documentation. It's something important that is not clear. Can you do these 2 changes if they make sense?